src/app/CommandlineArgumentExtractor.js

```diff
@@ -50,6 +50,8 @@ module.exports = class CommandlineArgumentExtractor {
      */
     get options() {
         return {
+            //Add the Cube loader (yet again)
+            applicationCubeURL:           this._get('--cubeloader-url'),
```

Comment:
The command line flags are more targeting developers, i don't think it is necessary to make the loading page configurable